beat/Functions/kinematic_routine_walking.m

```diff
@@ -92,9 +92,7 @@
 angle_label2=cat(2,angle_label(angle_r),angle_label(angle_l));
 
 %%save mROM value in .yaml file
-[aaa, name, extension]=fileparts(PlatformData);
-name2=regexprep(name,'_PlatformData','');
-file_id=fopen(strcat(outFolder,"/", name2, "_mROM", ".yaml"),'w'); %%open file to write into
+file_id = fopen(strcat(outFolder, "/pi_mrom.yaml"),'w'); %%open file to write into

```

Comment:
I am not sure that it the best solution for saving the pi. I mean, the mROM can be obtained with different protocol. Maybe it should be useful to put in the file name also the protocol number.
Another solution could be to put the name in the outFolder, but I believe that it is less automatic.

Reply:
The initial code made some assumption on the way the input filename wasstructured. We should not.
Your code generate a pi_mrom, we store it as is. It is up to the caller of your routines to make this relation. 

And actually, this is why we suggest to store all output in folder, previously created. This way, the caller can create folder protocol_i, and ask to store results there.